userdocs/src/usage/instance-selector.md

```diff
@@ -18,14 +18,14 @@ criteria passed to eksctl, run
 $ eksctl create cluster --instance-selector-vcpus=2 --instance-selector-memory=4
 ```
 
-This will create a cluster and nodegroup with the `instancesDistribution.instanceTypes` field set to
+This will create a cluster and a managed nodegroup with the `instanceTypes` field set to
 `[c5.large, c5a.large, c5ad.large, c5d.large, t2.medium, t3.medium, t3a.medium]` (the set of instance types returned may change).
 
 
-For managed nodegroups, the `instanceTypes` field will be set:
+For unmanaged nodegroups, the `instancesDistribution.instanceTypes` field will be set:
```

Comment:
This line says `unmanaged` nodegroups but the command below seems to create a managed nodegroups?

Reply:
You're right, the command below should have `--managed=false`.